### PR TITLE
Add publiccode.yml

### PR DESCRIPTION
**Pull request by m-kuhn**
This PR contains a proposal for a publiccode.yml file for QField.

The presence of a publiccode.yml file enables the listing of an open source project in the Italian Government FOSS catalogue (https://developers.italia.it/it/search?type=software_open&sort_by=relevance&page=0 ), which showcases FOSS software which can be useful for Public Administrations (as an example, it contains Libreoffice, too).

The publiccode.yml is also being proposed as an international standard for documenting FOSS software.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 724bfba93fb1b55d460e0736932311183d3f6a01:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-8286fca8b8144f86db4d904d415f3d6e6169cd20-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-8286fca8b8144f86db4d904d415f3d6e6169cd20-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-8286fca8b8144f86db4d904d415f3d6e6169cd20-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-8286fca8b8144f86db4d904d415f3d6e6169cd20-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 724bfba93fb1b55d460e0736932311183d3f6a01:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-85577f709dbfaf5c747ed213663f7acff0e31dbd-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-85577f709dbfaf5c747ed213663f7acff0e31dbd-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-85577f709dbfaf5c747ed213663f7acff0e31dbd-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-85577f709dbfaf5c747ed213663f7acff0e31dbd-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 7b393fa505ab1ecc6650ecac0da9b34969b7aab6:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-5ed1e525900186307c960cfe29691ef711c482e8-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-5ed1e525900186307c960cfe29691ef711c482e8-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-5ed1e525900186307c960cfe29691ef711c482e8-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-5ed1e525900186307c960cfe29691ef711c482e8-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for fdd7832d77a2af9f1d6a2575ae643e8d8fa913f4:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-0b64b028ba1f0ed979d9540be7e54208dfe05f88-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-0b64b028ba1f0ed979d9540be7e54208dfe05f88-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-0b64b028ba1f0ed979d9540be7e54208dfe05f88-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-0b64b028ba1f0ed979d9540be7e54208dfe05f88-x86.apk)

File: publiccode.yml [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+      and style new geometries from existing features)
+
+      ### 4. Collaborative work
+      
+      - Collaboratively work on datasets through [QFieldCloud](https://qfield.cloud)
+      
+    screenshots:
+      - https://github.com/opengisch/QField-docs/raw/master/en/images/qfield_picture.png
+    shortDescription: |-
+      QField is a full-featured, user-friendly, free-and-open-source mobile data collection
```
Comment (mbernasocchi) [inline publiccode.yml:93]: is the leading 
```suggestion
      QField is the leading full-featured, user-friendly, free-and-open-source mobile data collection
```

File: publiccode.yml [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    - environment
+    - government
+    - infrastructures
+    - local-authorities
+    - transportation
+    - tourism
+    - welfare
+it:
+  conforme:
+    gdpr: false
```
Comment (mbernasocchi) [inline publiccode.yml:110]: is this because of the cloud? or location services or?

**Review by mbernasocchi — APPROVED** [review]

File: publiccode.yml (hunk 2 above)
Comment (m-kuhn) [reply publiccode.yml:110]:
```suggestion
    gdpr: true
```

Was a copy paste from the qgis desktop proposal

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 8efa00727506b57cf2609228e91f5caf107c98c0:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-aecc281cbe0ed8def0e4fb79a4202d8a09998ed5-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-aecc281cbe0ed8def0e4fb79a4202d8a09998ed5-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-aecc281cbe0ed8def0e4fb79a4202d8a09998ed5-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1926-aecc281cbe0ed8def0e4fb79a4202d8a09998ed5-x86.apk)

**Pull request merged**